From 1aa9b8ada01ece155bbb6b7d0b447f5f5341d7ba Mon Sep 17 00:00:00 2001
From: Corentin Labbe <clabbe.montjoie@gmail.com>
Date: Mon, 12 Mar 2018 17:06:52 +0100
Subject: video: remove unused kconfig SH_LCD_MIPI_DSI
References: bsc#1087092
Patch-mainline: v4.16-rc4
Git-commit: 1aa9b8ada01ece155bbb6b7d0b447f5f5341d7ba

SH_LCD_MIPI_DSI is unused since commit 18b6562c243f ("fbdev: sh_mipi_dsi:
remove driver") so no need to keep it.

Fixes: 18b6562c243f ("fbdev: sh_mipi_dsi: remove driver")
Signed-off-by: Corentin Labbe <clabbe.montjoie@gmail.com>
Signed-off-by: Bartlomiej Zolnierkiewicz <b.zolnierkie@samsung.com>
Acked-by: Thomas Zimmermann <tzimmermann@suse.de>
---
 drivers/video/Kconfig | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/video/Kconfig b/drivers/video/Kconfig
index 3c20af999893..429f15b2b51a 100644
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -8,9 +8,6 @@ menu "Graphics support"
 config HAVE_FB_ATMEL
 	bool

-config SH_LCD_MIPI_DSI
-	bool
-
 source "drivers/char/agp/Kconfig"

 source "drivers/gpu/vga/Kconfig"
--
2.18.0

